[ews-build.webkit.org] Convert RemoveFlagsOnPatch to Twisted asynchronous requests

JonWBedard opened the pull request:
#### 31ea862d60c26b6c9f319f02622bbaa48f7ae2a0
<pre>
[ews-build.webkit.org] Convert RemoveFlagsOnPatch to Twisted asynchronous requests
<a href="https://bugs.webkit.org/show_bug.cgi?id=251281">https://bugs.webkit.org/show_bug.cgi?id=251281</a>
rdar://104752905

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(BugzillaMixin.remove_flags_on_patch): Convert to Twisted asynchronous requests.
(RemoveFlagsOnPatch.run): Renamed from start.
(RemoveFlagsOnPatch.start): Renamed to run.

Canonical link: <a href="https://commits.webkit.org/259728@main">https://commits.webkit.org/259728@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e0e10ea4305ea43f180bc12e0916d46409bb73c2

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/105056 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/14135 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/37946 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/114319 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/174503 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/108959 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/15273 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/5057 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/97375 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/113332 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/110812 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/11811 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/94808 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/39312 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/93679 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/26435 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/80977 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/7473 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/27794 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/7567 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/4378 "Passed tests") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/20/builds/103855 "Passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/13622 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/47347 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/9356 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/3600 "Built successfully and passed tests") | | | | 
<!--EWS-Status-Bubble-End-->

aj062 reviewed (approved):
r+ assuming it's tested

JonWBedard committed:
[ews-build.webkit.org] Convert RemoveFlagsOnPatch to Twisted asynchronous requests
https://bugs.webkit.org/show_bug.cgi?id=251281
rdar://104752905

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(BugzillaMixin.remove_flags_on_patch): Convert to Twisted asynchronous requests.
(RemoveFlagsOnPatch.run): Renamed from start.
(RemoveFlagsOnPatch.start): Renamed to run.

Canonical link: https://commits.webkit.org/259728@main
